go.mod

```diff
@@ -1,15 +1,15 @@
-module github.com/crossplane/oam-controllers
+module github.com/crossplane/addon-oam-kubernetes-local
```

Comment:
Changing this means we must change repos which rely on this module. We'd better consider using a better name if we want to do this. What about `oam-core` ？

Reply:
yeah, better to keep this name

Reply:
Done. Kept the same name.